app/api/index.js

```diff
@@ -1,10 +1,10 @@
-function fetchNodes(limit = 5, offset = 0, sort = 'id', order = 'desc') {
+function fetchNodes(limit = 10, offset = 0, sort = 'id', order = 'desc') {
```

Comment:
It's used more than once maybe it will be better to take the value outside to the const?